cli/src/migrations/files.ts

```diff
@@ -32,7 +32,7 @@ export async function readMigrationsDir() {
 
 export type LocalMigrationFile =
   | Schemas.MigrationObject
-  | (Schemas.PgRollMigrationHistoryItem & { id?: never; checksum?: never; operations?: never[] });
+  | (LocalMigrationFilePgroll & { id?: never; checksum?: never; operations?: never[] });
```

Comment:
I find confusing that we have now 2 types called very similar `LocalMigrationFile` and `LocalMigrationFilePgroll`. I understand the naming but it could be difficult to follow in a few months and reason why we needed two types. Maybe they can be combined together in one or we find a different name. WDYT?

Reply:
Yeah agree, I tried to simplify it.